airflow/www/views.py

```diff
@@ -2960,6 +2960,9 @@ class ConnectionModelView(wwwutils.SuperUserMixin, AirflowModelView):
         'extra__google_cloud_platform__key_path',
         'extra__google_cloud_platform__keyfile_dict',
         'extra__google_cloud_platform__scope',
+        'extra__grpc__auth_type',
```

Comment:
please add these changes to `airflow/www_rbac/views.py` as well, thanks!

Reply:
I tried it but it somehow lead to test failure of the BaseModelView which I think shouldn't matter. Rolled back it for now